src/lib/views/messages/RenderKind1.svelte

```diff
@@ -11,10 +11,13 @@
 	import { viewed } from '@/snort_workers/main';
 	import UserDisplayName from '@/components/UserDisplayName.svelte';
 	import UserProfilePic from '@/components/UserProfilePic.svelte';
-	import { formatTimeAgo } from '../../utils'
+	import { formatTimeAgo } from '@/utils'
+	import { currentPubkey } from '@/stores/user'
 
 	export let note: NostrEvent;
-	export let onClickReply: () => void;
+	export let onClickReply = () => {
+		currentPubkey.set(note.pubkey);
```

Comment:
We can't rely on this because it is overwritten by the parent.

Reply:
Progress @gsovereignty  

Time spent Log.
initial environment set up & research (10hrs) 
Development and testing (6hrs) 
Debugging (6hrs) 
Code review and addressing feedback (5hrs) 

TOTAL: 27hrs